src/metrax/metrics.py

```diff
@@ -284,7 +366,29 @@ def compute(self) -> jax.Array:
 
 @flax.struct.dataclass
 class AUCPR(clu_metrics.Metric):
-  """Computes area under the precision-recall curve for binary classification given `predictions` and `labels`.
+  r"""Computes area under the precision-recall curve for binary classification given `predictions` and `labels`.
+
+  The Precision-Recall curve shows the tradeoff between precision and recall at different
+  classification thresholds. The area under this curve (AUC-PR) provides a single score
+  that represents the model's ability to distinguish between classes across all possible
+  thresholds.
```

Comment:
Can this have slightly more detail to distinguish it from the docstring in AUCROC? Maybe like:

`... provides a single score that represents the model's ability to identify positive cases across all possible classification thresholds, particularly in imbalanced datasets.`